package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/ui-component-library",
-  "version": "0.2.6",
+  "version": "0.2.8",
```

Comment:
last published version is `0.2.8` so needs to be higher if we want this to publish

Reply:
done